huxley/core/middlewares.py

```diff
@@ -1,24 +1,34 @@
 # Copyright (c) 2011-2015 Berkeley Model United Nations. All rights reserved.
 # Use of this source code is governed by a BSD License (see LICENSE).
 
-import json, logging
+import json, logging, sys, traceback
+
+from huxley.settings.main import DEBUG
```

Comment:
```python
from django.conf import settings
```
and then `settings.DEBUG` in code: https://docs.djangoproject.com/en/1.8/topics/settings/#using-settings-in-python-code